drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -123,7 +123,7 @@
         </div>
         <div class="row">
           <label for="item-{{id}}-numerical-margin">
-            {{i18n "Margin ± (when a numerical value is required, values entered by students must not differ from the expected value by more than this margin; default is zero)"}}
+            {{i18n "Margin +/- (when a numerical value is required, values entered by students must not differ from the expected value by more than this margin; default is zero)"}}
```

Comment:
@bradenmacdonald 

After replacing `load_resource` with `ResourceLoader.load_unicode` in the [line](https://github.com/open-craft/xblock-drag-and-drop-v2/blob/add-theming/drag_and_drop_v2/drag_and_drop_v2.py#L180) that pulls in `js_templates.html` for the EDIT dialog, I got the following error:

    Error: 'ascii' codec can't decode byte 0xc2 in position 5557: ordinal not in range(128)

The difference between `load_resource` and `load_unicode` from xblock-utils is that the former wasn't calling `unicode` on the content pulled from a resource before returning it.

Changing the symbol seemed like a better option than keeping `load_resource` just for the sake of avoiding this error, but let me know -- maybe there's another way to approach this?